kratos/tests/cpp_tests/geometries/test_quadrilateral_3d_4.cpp

```diff
@@ -111,8 +111,7 @@ namespace Testing
     {
         auto geom = GenerateRightQuadrilateral3D4<Node<3>>();
         
-        KRATOS_CHECK_NEAR(geom->Area(), 1.06947235, TOLERANCE);
-//         KRATOS_CHECK_NEAR(geom->Area(), 1.08935, TOLERANCE); // NOTE: Solution from Mathematica
+        KRATOS_CHECK_NEAR(geom->Area(), 1.06948, 1.0e-5);
```

Comment:
which one is the "correct" solution? 

We should test this with very many decimals ...

Reply:
I did this PR long time ago, let me check in any case

Reply:
Related to #4305.

Reply:
the point is that this should have very many decimals (around 15) since it is around 1.0

Reply:
I agree with @RiccardoRossi 's observation. The tolerance in this case must be smaller.

Reply:
why the tolerance and results are changed?